src/main/java/bio/terra/janitor/db/exception/DuplicateDbKeyException.java

```diff
@@ -2,10 +2,7 @@
 
 import bio.terra.janitor.common.exception.BadRequestException;
 
-/**
- * Exception when inserting into tracked_resource table while there is already a row with the same
- * key.
- */
+/** Exception when inserting postgres table while there is already a row with the same key. */
 public class DuplicateDbKeyException extends BadRequestException {
```

Comment:
Like I asked in previous round of comments.  Do you think it make sense if we keep it as it may be used for future Dao's use cases? Or we should delete it for now and added it back if we need?

Reply:
Deleted

Reply:
YAGNI. if we don't need it yet, don't add it yet.